web/src/main/java/org/kontinuity/catapult/web/api/CatapultResource.java

```diff
@@ -221,34 +208,11 @@ private Response fling(Projectile projectile) {
         return Response.temporaryRedirect(consoleOverviewUri).build();
     }
 
-    private class ProjectileOrResponse {
-        ProjectileOrResponse(Projectile projectile) {
-            this(projectile, null);
-        }
-
-        ProjectileOrResponse(Response response) {
-            this(null, response);
-        }
-
-        private ProjectileOrResponse(Projectile projectile, Response response) {
-            this.projectile = projectile;
-            this.response = response;
-        }
-
-        private final Projectile projectile;
-
-        private final Response response;
-
-        boolean hasResponse() {
-            return response != null;
-        }
-
-        Projectile getProjectile() {
-            return projectile;
-        }
-
-        Response getResponse() {
-            return response;
-        }
+    private CreateProjectile createCreateProjectile(String path, String gitHubAccessToken) {
```

Comment:
But isn't that the same as calling this method twice? I thought the Builder would be a replacement for factory methods